packages/devtools_app/test/test_infra/flutter_test_environment.dart

```diff
@@ -39,12 +37,12 @@ class FlutterTestEnvironment {
     return flutterExe.isNotEmpty ? flutterExe : defaultFlutterExecutable;
   }
 
-  FlutterRunConfiguration _runConfig;
-  FlutterRunConfiguration get runConfig => _runConfig;
-  FlutterRunTestDriver _flutter;
-  FlutterRunTestDriver get flutter => _flutter;
-  VmServiceWrapper _service;
-  VmServiceWrapper get service => _service;
+  FlutterRunConfiguration? _runConfig;
+  FlutterRunConfiguration? get runConfig => _runConfig;
+  FlutterRunTestDriver? _flutter;
+  FlutterRunTestDriver? get flutter => _flutter;
+  VmServiceWrapper? _service;
```

Comment:
_service can be late and service getter can be non-nullable

Reply:
done